src/scss/index.scss

```diff
@@ -1 +1,131 @@
+@import 'variables';
+
+$dark: #eab5c0;
+
+@import '../../node_modules/bootstrap/scss/bootstrap.scss';
+
+@mixin border-left-radius($radius) {
+    border-top-left-radius: $radius;
+    border-bottom-left-radius: $radius;
+}
+
+
+.login-body {
+    background-color: $dark
+}
+
+.login-container {
+
+    width: 100%;
+    height: 100vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+
+    @media (max-width: 767px) {
+        height: auto;
+    }
+
+    .login-inner {
+        width: 600px;
+       
+        background-color: #FFFFFF;
+        border-radius: 10px;
+
+        img {
+
+
+            
+
+            @media (min-width: 768px) {
+               @include border-left-radius(10px);
+                width: 100%;
+                height: 100%;
+              
+            }
+
+            width: 100%;
+            object-fit: cover;
+            object-position: center;
+            height: 200px;
+           
+
+            @media (max-width: 767px) {
+                border-radius: 10px;
+            }
+          
+         
+        }
+    }
+   
+  
+
+}
+
+
+.image-container {
+    width: 150px;
+    height: 150px;
+    border-radius: 50%;
+    //margin: 0px auto;
+    overflow: hidden;
+
+    img {
+        width: 100%;
+        height: 100%;
+        object-fit: cover;
+        object-position: center;
+    }
+}
+
+
+.followers-list {
+   
+    li {
+        list-style: none;
+
+        .follower-image-container {
+            width: 50px;
+            height: 50px;
+            border-radius: 50%;
+            background-color: #eab5c0;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+        }
+
+        .follower-image-content {
+            display: flex;
+            flex-direction: column;
+          
+            p {
+                text-align: left;
```

Comment:
https://getbootstrap.com/docs/5.3/utilities/text/

Reply:
Thank you Oliver. i will fix it all.